# 飞机基础油门值
BaseThrust: 0.525
ThrustLimit: [0.280, 0.580]


# 位置环 PID 参数
posKP: [3, 1, 4]
# posKI: [0.05, 0.05, 0.2] 
posKI: [1.0, 0.0, 1.0]
posKD: [0.0, 0.0, 0.0]
# 速度环 PID 参数
velKP: [3, 1.5, 4]
velKI: [1.0, 0.0, 1.0]
velKD: [0.0, 0.0, 0.0]
